iib/workers/s3_utils.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+
+import boto3
+
+from iib.exceptions import IIBError
+from iib.workers.config import get_worker_config
+
+log = logging.getLogger(__name__)
+
+conf = get_worker_config()
+
+
+def upload_file_to_s3_bucket(file_path, key_prefix, file_key):
+    """
+    Upload artifact file to AWS S3 bucket.
```

Comment:
Why do we have `conf` outside the function? 

Wouldn't it be better to put it inside? I saw somewhere that `get_worker_config` can lead to some circular imports, so if no other function in this file will use it maybe it's better to put it inside.

BTW, thanks for creating a separate utils for S3!